webpack.config.shared.js

```diff
@@ -57,7 +57,11 @@ module.exports = {
     exclude: /node_modules/,
     loader: 'babel-loader',
     query: {
-      "presets": ["latest", "react"],
+      "presets": [
+        ["env", { "modules": false }],
```

Comment:
Would tree shaking be turned on by this PR? Is tree shaking broken in a way that would affect us?